llama_stack/distribution/datatypes.py

```diff
@@ -146,6 +147,14 @@ class StackRunConfig(BaseModel):
 a default SQLite store will be used.""",
     )
 
+    # registry of "resources" in the distribution
+    models: List[Model] = Field(default_factory=list)
+    shields: List[Shield] = Field(default_factory=list)
+    memory_banks: List[MemoryBank] = Field(default_factory=list)
+    datasets: List[Dataset] = Field(default_factory=list)
+    scoring_fns: List[ScoringFn] = Field(default_factory=list)
```

Comment:
When we run `llama stack build`, we will use generate a template `StackRunConfig` with the default values. Do we have a reference `run.yaml` that user are expected to start with?